pkg/executor/importer/import.go

```diff
@@ -816,7 +818,23 @@ func (p *Plan) initParameters(plan *plannercore.ImportInto) error {
 	}
 	for _, opt := range plan.Options {
 		if opt.Value != nil {
-			val := opt.Value.StringWithCtx(evalCtx, errors.RedactLogDisable)
+			var val string
+			if constValue, ok := opt.Value.(*expression.Constant); ok {
+				// we can stringify 'constant' directly because
+				// redact is set to 'RedactLogDisable' by default
+				switch constValue.Value.Kind() {
+				case types.KindString, types.KindBytes:
+					val = constValue.Value.GetString()
+				case types.KindMysqlJSON:
+					val = constValue.Value.GetMysqlJSON().String()
+				case types.KindVectorFloat32:
+					val = constValue.Value.GetVectorFloat32().String()
```

Comment:
@breezewish The option here is unlikely to process vectors. So maybe we can remove it ?